src/mode.js

```diff
@@ -51,18 +51,19 @@ export function setModeForTesting(m) {
  * @return {!ModeDef}
  */
 function getMode_() {
-  const isLocalDev = (location.hostname == 'localhost' ||
-      (location.ancestorOrigins && location.ancestorOrigins[0] &&
-          location.ancestorOrigins[0].indexOf('http://localhost:') == 0)) &&
+  const hostname = location.hostname;
```

Comment:
Could you file an issue for correctly picking up dev mode in 3p iframes? Thanks!